dev_guide/volumes.adoc

```diff
@@ -306,3 +314,38 @@ List volume *v1* defined on all deployment configurations:
 ----
 $ oc volume dc --all --name=v1
 ----
+
+[[volumes-specifying-a-subpath]]
+== Specifying a Sub-path
+
+Use the `volumeMounts.subPath` property to specify a `subPath` inside a volume
+instead of the volume's root. `subPath` allows you to share one volume for
+multiple uses in a single pod.
+
+.Example subPath Usage
```

Comment:
The only suggestion I have here is add a preamble showing the contents of this volume.